src/components/PeopleManagement/LearnerDetailPage/LearnerDetailGroupMemberships.tsx

```diff
@@ -87,6 +87,3 @@ LearnerDetailGroupMemberships.propTypes = {
 };
 
 export default LearnerDetailGroupMemberships;
-
```

Comment:
nit: extra lint comments

Reply:
These comments are being taken away with this change